test/server/lib/OIDCConfig.ts

```diff
@@ -182,7 +188,7 @@ describe('OIDCConfig', () => {
           Object.assign(process.env, ctx.env);
           const client = new ClientStub();
           client.issuer.metadata.end_session_endpoint = ctx.end_session_endpoint;
-          const promise = OIDCConfigStubbed.build(client.asClient());
+          const promise = OIDCConfigStubbed.buildWithStub(client.asClient());
```

Comment:
Here `const client` seems to be not used, you can call ouside this line. So you can directly use `buildWithStub()`

Reply:
Not here (the line above changes some property of the client) but it is true for other places.

Thanks, that's done!